plugins/modules/postgresql_user.py

```diff
@@ -154,6 +154,20 @@
     type: bool
     default: true
     version_added: '0.2.0'
+  configuration:
+    description:
+      - Role-specific configuration parameters that would otherwise be set by C(ALTER ROLE user SET variable TO value;).
+      - Takes a list of strings like C(key=value) that is split on the first equal-sign.
+      - Sets or updates any parameter in the list that is not present or has the wrong value in the database.
+      - Removes any parameter from the user that is not listed here.
+      - If the first item is 'purge', all configuration parameters will be removed from the user.
+      - If omitted or empty, the current parameters of the user will not be changed.
+      - Using this will leave the I(user) as well as this parameter open to SQL-injections,
+        parameters are checked if I(trust_input) is C(false).
+    type: list
+    elements: str
+    default: []
+    version_added: '4.0.0'
```

Comment:
```suggestion
    version_added: '3.5.0'
```
it's a minor change as it doesn't break the interface